Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -200,6 +200,16 @@ extension BrowserViewController: BrowserToolbarDelegate {
         controller.tabManager = tabManager
         presentViewController(controller, animated: true, completion: nil)
     }
+
+    func didClickShare() {
+        if let selected = tabManager.selectedTab {
+            if let url = selected.url {
+                var vc = UIActivityViewController(activityItems: [selected.title ?? url, url], applicationActivities: nil)
```

Comment:
I don't think the `selected.title ?? url` bit is correct. If it is good practice to attach the URL if a title is missing (i don't know if it is, we should check what safari does probably) then we should attach `url.absoluteString` instead. Otherwise we are just attaching two similar items of type `public.url` and the receiving extension would not see the difference between them.